### PR TITLE
PMM-9907 added support for Ubuntu 22.04 LTS

### PR DESCRIPTION
**Pull request by rasika-chivate**
Added Support for Ubuntu 22.04 LTS for PMM Client

### CODE REVIEW

File: docs/setting-up/client/index.md [hunk 1]
```diff
@@ -495,3 +496,4 @@ pmm-admin remove <service-type> <service-name>
 [Ubuntu 16.04 (Xenial Xerus)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/xenial/
 [Ubuntu 18.04 (Bionic Beaver)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/bionic/
 [Ubuntu 20.04 (Focal Fossa)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/focal/
+[Ubuntu 22.04 (Jammy Jellyfish)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/jammy/
```
Comment (nikita-b) [inline docs/setting-up/client/index.md:499]: We need to remove Xenial (We don't support it anymore) + we need to add Debian 11.

**Review by nikita-b — APPROVED** [review]

File: docs/setting-up/client/index.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/setting-up/client/index.md:499]: ok

Comment (rasika-chivate) [reply docs/setting-up/client/index.md:499]: @nikita-b should we include Ubuntu 11.10 (Oneiric Ocelot)

Comment (nikita-b) [reply docs/setting-up/client/index.md:499]: Nope. We support only three Ubuntu:
Bionic, Focal, Jammy

File: docs/setting-up/client/index.md [hunk 2]
```diff
@@ -248,9 +248,10 @@ Here are the download page links for each supported platform.
 - [Debian 10 (Buster)]
 - [Red Hat/CentOS/Oracle 7]
 - [Red Hat/CentOS/Oracle 8]
-- [Ubuntu 16.04 (Xenial Xerus)]
+- [Ubuntu 11.10 (Oneiric Ocelot)]
```
Comment (denisok) [inline docs/setting-up/client/index.md:251]:
```suggestion
```

File: docs/setting-up/client/index.md [hunk 3]
```diff
@@ -492,6 +493,7 @@ pmm-admin remove <service-type> <service-name>
 [PMM Client Docker image]: https://hub.docker.com/r/percona/pmm-client/tags/
 [Red Hat/CentOS/Oracle 7]: https://www.percona.com/downloads/pmm2/{{release}}/binary/redhat/7/
 [Red Hat/CentOS/Oracle 8]: https://www.percona.com/downloads/pmm2/{{release}}/binary/redhat/8/
-[Ubuntu 16.04 (Xenial Xerus)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/xenial/
+[Ubuntu 11.10 (Oneiric Ocelot)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/oneiric/
```
Comment (denisok) [inline docs/setting-up/client/index.md:496]:
```suggestion
```

**Review by denisok — CHANGES_REQUESTED** [review]

**Review by denisok — APPROVED** [review]

**Review by nikita-b — APPROVED** [review]

**Review by nikita-b — APPROVED** [review]

**Comment by rasika-chivate** [comment]
> I believe this should be Debian 11 "bullseye"

Yes. Updated it.

File: docs/setting-up/client/index.md [hunk 4]
```diff
@@ -481,6 +482,7 @@ pmm-admin remove <service-type> <service-name>
     - [PMM Client architecture](../../details/architecture.md#pmm-client)
     - Thanks to [paskal] for original Docker compose files
 
+[Debian 11 Bullseye)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/bullseye/
```
Comment (denisok) [inline docs/setting-up/client/index.md:485]:
```suggestion
[Debian 11 (Bullseye)]: https://www.percona.com/downloads/pmm2/{{release}}/binary/debian/bullseye/
```

**Review by catalinaadam — APPROVED** [review]

**Pull request merged**